include/lsst/jointcal/Associations.h

```diff
@@ -44,12 +46,27 @@ class Associations {
             : _commonTangentPoint(Point(std::numeric_limits<double>::quiet_NaN(),
                                         std::numeric_limits<double>::quiet_NaN())) {}
 
+    /**
+     * Create an Associations object from a pre-built list of ccdImages.
+     *
+     * @param imageList A pre-built ccdImage list.
+     */
+    Associations(CcdImageList const &imageList)
+            : ccdImageList(imageList),
+              _commonTangentPoint(Point(std::numeric_limits<double>::quiet_NaN(),
+                                        std::numeric_limits<double>::quiet_NaN())) {}
+
     /// No moves or copies: jointcal only ever needs one Associations object.
     Associations(Associations const &) = delete;
     Associations(Associations &&) = delete;
     Associations &operator=(Associations const &) = delete;
     Associations &operator=(Associations &&) = delete;
 
+    /**
+     * Sets a shared tangent point for all ccdImages, using the mean of the centers of all ccdImages.
+     */
+    void setCommonTangentPoint();
```

Comment:
Consider a name such `computeCommonTangentPoint`. It is less confusing than a `set` method that takes no arguments (usually `set` takes the value to be set). Plus less overloading makes the Python easier to understand.